galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,210 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from random import sample
+from rest_framework.response import Response
+from pulp_ansible.app.models import CollectionVersion, AnsibleDistribution
+from galaxy_ng.app.models import Namespace
+from galaxy_ng.app import settings
+from galaxy_ng.app.api import base as api_base
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request, *args, **kwargs):
+        golden_name = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
+
+        distro = AnsibleDistribution.objects.get(base_path=golden_name)
+        repository_version = distro.repository.latest_version()
+        versions = CollectionVersion.objects.filter(pk__in=repository_version.content)
+        collection_count = versions.select_related("collection").count()
+
+        partner_count = Namespace.objects.count()
+
+        namespace = sample(list(Namespace.objects.all()), 1)[0]
+
+        data = {
+            "estate": {
+                "items": [
+                    {
+                        "id": "ansible-collections",
+                        "count": collection_count,
+                        "shape": {
+                            "title": "Collections",
+                            "href": "./ansible/automation-hub/",
+                        },
+                        "permissions": [
+                            {
+                                "method": "isEntitled",
+                                "args": ["ansible"],
+                            }
+                        ],
+                    },
+                    {
+                        "id": "ansible-partners",
+                        "count": partner_count,
+                        "shape": {
+                            "title": "Partners",
+                            "href": "./ansible/automation-hub/partners/",
+                        },
+                        "permissions": [
+                            {
+                                "method": "isEntitled",
+                                "args": ["ansible"],
+                            }
+                        ],
+                    },
+                ],
+            },
+            "recommendations": {
+                "recs": [
+                    {
+                        "id": "ansible-partner",
+                        "icon": "bulb",
+                        "action": {
+                            "title": "Check out our partner %s" % namespace.name,
```

Comment:
I'd recommend `namespace.company` here. That will show the more human readable name instead of the internal namespace value.